### ISSUE TITLE
[Feature request] ADD support for `eslint-html-parser`

### ISSUE THREAD

**Issue by francoismassart** [francoismassart/eslint-plugin-tailwindcss#31]
The default parser can report unrelated JSX errors for regular html markup like:

```html
<div>1</div>
<div>2</div>
```
reporting `Parsing error: Adjacent JSX elements must be wrapped in an enclosing tag`

We should offer the possibility to use `eslint-html-parser`

**Comment by francoismassart** [comment]
I'll probably go with @angular-eslint/template-parser
See #85
